Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Entity/EntityFunctions.h

```diff
@@ -30,3 +30,5 @@ namespace ScriptCanvas
         AZStd::string ToString(const AZ::EntityId& source);
     } // namespace EntityFunctions
 } // namespace ScriptCanvas
+
+#include <Include/ScriptCanvas/Libraries/Entity/EntityFunctions.generated.h>
```

Comment:
nitpick: missing newline at end of file
```suggestion
#include <Include/ScriptCanvas/Libraries/Entity/EntityFunctions.generated.h>

```